apps/authentication-service/.env.example

```diff
@@ -0,0 +1,19 @@
+# Port of the service
+PORT=8089
+
+# Environment (dev, prod)
+ENV=dev
+
+# Token Cache filename
+TOKEN_CACHE_FILE=brkn-arrw.txt
+
+# Base URL of the Client (Web UI)
+CLIENT_ORIGIN_URL=http://localhost:3031
+
+# Auth0 parameters, found in your Auth0 dashboard.
+AUTH0_DOMAIN=
```

Comment:
Don't we want Domain, audience, and client ID to be in here? They're not sensitive right?

Reply:
Default dev values have been added to the example file.